airrohr-firmware/airrohr-firmware.ino

```diff
@@ -3457,6 +3468,48 @@ void fetchSensorSPH0645(String& s){
 
 }
 
+/*****************************************************************
+ * Initialize RTC                                        		 *
+ *****************************************************************/
+void init_RTC()
+{
+	pinMode(RTC_PIN_SDA, OUTPUT);
+	pinMode(RTC_PIN_SCL, OUTPUT);
+	if (!rtc.isrunning())
+	{
+		// sets time to the date this sketch was compiled
+		// rtc.adjust(DateTime(F(__DATE__), F(__TIME__)));
+		// Set explicit time for example 19th March 2020 at 12 noon.
+		// rtc.adjust(DateTime(2020, 5, 19, 12, 00, 00));
+	}
+}
+
+/*****************************************************************
+ * read RTC sensor values                                        *
+*****************************************************************/
+static void fetchSensorRTC(String &s) {
+	debug_outln_verbose(FPSTR(DBG_TXT_START_READING), "RTC");
+
+	DateTime now = rtc.now();
+
+	if(send_now) {
```

Comment:
@Warukira this seems to be a repeat of what's happening in a previous function. Good for debugging but not useful.

Reply:
@chegejames , was using it for debugging, but sorting it out.